lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -1959,7 +1963,9 @@ def _add_pebble_layer(self, workload_binary_path: str) -> None:
                 }
             },
         }
-        self._container.add_layer(self._container_name, pebble_layer, combine=True)
+        self._container.add_layer(
+            self._container_name, pebble_layer, combine=True  # pyright: ignore
```

Comment:
Just curious why the ignore here?

Reply:
The pebble layer should be of `LayerDict` type, but I wanted this PR to change as little as possible to avoid breaking anything